### DISCUSSION TITLE
Specter (Saltedge) connection not automatically fetching new AMEX transactions

### DISCUSSION THREAD

**Discussion by ipsi** [firefly-iii/firefly-iii#7942] [Firefly III] [answered]
I'm _pretty sure_ this is a limitation of exactly how Spectre is grabbing the transactions and not something that can be worked around but I would like to double-check.

I have an American Express Germany account, and I'm trying to import transactions from it using the Firefly Data Importer. It worked fine yesterday, but it hasn't picked up any new transactions for today, which it should have (manually importing via the Firefly UI both times). Looking at the Spectre page for that particular connection, it displays this:

![grafik](https://github.com/firefly-iii/firefly-iii/assets/667984/80b97b46-5aaf-402b-a29a-26a5069d5ca7)

The "InteractiveAdapterTimeout" error code, I'm fairly certain, means that AMEX asked me to choose how I wanted my second factor provided (email or text message), but obviously couldn't since it wasn't interactive - if I hit the "refresh" button on the Spectre site, it would do exactly that. I suppose I could set it up to refresh transaction every 4 minutes or so and see if that keeps the session alive, but that feels like a bit of a dodgy hack.

Is there anything I can reasonably do here, or would it be less of a headache to just manually input transactions?

**Comment by JC5** [comment, marked as answer]
Check this with Spectre as well, but I'm afraid you're out of luck here.
